src/Microsoft.DotNet.Metrics/Microsoft.DotNet.Metrics.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <TargetLatestRuntimePatch>False</TargetLatestRuntimePatch>
```

Comment:
This should also not be necessary. It may have been necessary where this was copied from, but this library should not need it.